src/chunk.c

```diff
@@ -3240,6 +3245,15 @@ ts_chunk_drop_process_materialization(Oid hypertable_relid,
 /* Continuous agg materialization hypertables can be dropped
  * only if a user explicitly specifies the table name
  */
+Datum
+ts_pg_chunk_drop(PG_FUNCTION_ARGS)
+{
+	int32 chunk_id = PG_GETARG_INT32(0);
```

Comment:
Bad cast. This is a regclass, not int32.

Reply:
Fixed.